EssentialEstablishmentGenerator/PlotHook/PlotHookPaper.twee

```diff
@@ -1,3 +1,3 @@
 :: PlotHookPaper
-You see a noticeboard next to the $tavern.wordNoun's $tavern.material bar. Walking over to it, you see a number of barter requests nailed to the noticeboard; simple things like 'have chook, want grain', 'have grain, want chook'.
+You see a noticeboard next to the $tavern.wordNoun's $tavern.material bar. Walking over to it, you see a number of barter requests nailed to the noticeboard; simple things like <<print either ("'have chook, want grain, have grain, want chook'", "'need an exterminator'", "'need help with rat problems'", "'hungry, don't want to walk'", "'need my roof cleaned'", "'I want an apple'", "'need a date for an upcoming wedding'")>>.
```

Comment:
`either()` does the same thing as `<<print [].seededrandom()>>`, so it might be good for code uniformity to change that; it's somewhat unclear what `either()` does to people unfamiliar with Twine notation, because it doesn't use brack notation, making it look like a function. We can also stop doubling up on `'`s by adding them on the outside of it, i.e. `simple things like '<<print [].seededrandom()>>'.`